Added Alert component

sanathkumar26700 opened the pull request:
Three types of Alerts have been added,

1. Standard alert
2. Standard alert with icons
3. Alert with a functional close button

sanathkumar26700 committed:
Add styling to alert components

sanathkumar26700 committed:
Add alert documentation site

kiranpratyush commented on Components/Alert component/alert.css:
```diff
@@ -0,0 +1,60 @@
+    right: 0;
+    opacity: 0.5;
+    cursor: pointer;
+    outline: none;
```
We can try looking into it. Outline is not recommended to be none because of accessibility reason I think

kiranpratyush commented on Components/Alert component/alert.html:
```diff
@@ -0,0 +1,129 @@
+
+</body>
+<script>
+    const alertExampleBox = document.querySelector("#alert-close");
```
We can make the separate script.js file to add scripts

sanathkumar26700 committed:
Refactor: remove none property of outline

For accessibility reasons outline property should not be none, hence removed it.

sanathkumar26700 committed:
fix : Correct the link for alert button

srishti-maurya commented on Components/Alert component/alert.css:
```diff
@@ -0,0 +1,59 @@
+
+.alert-warning {
+    color: #ff9204;
+    background-color: #ffe7c5;
```
maybe you may use color variables here
